include/sfx.h

```diff
@@ -243,7 +243,7 @@ typedef enum { MONO_SOUND, STEREO_SOUND } soundMode;
 #define SFX_UNK_6A3 0x6A3
 #define NA_SE_EV_CLOCK_TICK 0x6A1
 #define REBOUND_STONE_BOUNCE 0x6A4
-#define NA_SE_PL_COLLECT_GOLD 0x6A9
+#define SFX_COLLECT_GOLD 0x6A9
```

Comment:
Looks like there's a previous reference that also needs to be updated: https://github.com/Xeeynamo/sotn-decomp/blob/1d53f6e7152d2353edd069a65bf671f952b8f5d4/src/st/collect_gold.h#L8

Reply:
Oh wow, I can't believe I missed that.